src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -250,7 +248,11 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
     }
 
     private get width() {
-        return 220 + this.props.data.labels.length * 57;
+        const maxWidth = 220 + this.props.data.labels.length * 57;
+        if (document.getElementById("pancancer_study_summary")) {
```

Comment:
hmm. why is there an element of Id pancancer_study_summary?  lets think of a way to do this without cloodge.   like, lets give the chart a max width of whatever the fixed width of the content container of site is.  
we could measure this, but lets do this kind of "dirty" thing in the page component and pass it in through props to the this component.   in general, keep these presentation components "clean."  They shouldn't have to know about their environment.